http/translator_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"io"
 	"io/ioutil"
 	gohttp "net/http"
+	"sync/atomic"
```

Comment:
instead of using sync/atomic, I think it'd be more idiomatic (and less prone to confusion) to use a channel.

Then, at the bottom, instead of sleeping for 100ms and checking, you can use a `select` to receive from the channel, or a 100ms timer. That way the test ends as soon as the channel is closed rather than waiting the full 100ms every time.